heft-plugins/heft-webpack4-plugin/heft-plugin.json

```diff
@@ -5,7 +5,16 @@
     {
       "pluginName": "Webpack4Plugin",
       "entryPoint": "./lib/Webpack4Plugin",
-      "optionsSchema": "./lib/schemas/heft-webpack4-plugin.schema.json"
+      "optionsSchema": "./lib/schemas/heft-webpack4-plugin.schema.json",
+
+      "parameterScope": "webpack4",
+      "parameters": [
+        {
+          "longName": "--serve",
+          "parameterKind": "flag",
+          "description": "Start a local web server for testing purposes using webpack-dev-server.  This parameter is only available when running in watch mode."
```

Comment:
Serve used to imply `--watch`. Having to always specify `--serve --watch` is a slightly inconvenient.

Reply:
It's because `--watch` is no longer a flag, it's the whole action, ie. `heft build-watch --serve` would be the command.